2022-02-02 17:33:32,890 INFO    MainThread:8677 [wandb_setup.py:_flush():75] Loading settings from /home/yogender/.config/wandb/settings
2022-02-02 17:33:32,890 INFO    MainThread:8677 [wandb_setup.py:_flush():75] Loading settings from /home/yogender/Desktop/KaggleHousePricePrediction/RavirajMlOps/MLOps-Basics/week_1_wandb_logging/wandb/settings
2022-02-02 17:33:32,891 INFO    MainThread:8677 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-02-02 17:33:32,891 INFO    MainThread:8677 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'week_1_wandb_logging/train.py', 'program': 'train.py'}
2022-02-02 17:33:32,891 INFO    MainThread:8677 [wandb_init.py:_log_setup():386] Logging user logs to /home/yogender/Desktop/KaggleHousePricePrediction/RavirajMlOps/MLOps-Basics/week_1_wandb_logging/wandb/run-20220202_173332-18e8rx8m/logs/debug.log
2022-02-02 17:33:32,892 INFO    MainThread:8677 [wandb_init.py:_log_setup():387] Logging internal logs to /home/yogender/Desktop/KaggleHousePricePrediction/RavirajMlOps/MLOps-Basics/week_1_wandb_logging/wandb/run-20220202_173332-18e8rx8m/logs/debug-internal.log
2022-02-02 17:33:32,893 INFO    MainThread:8677 [wandb_init.py:init():420] calling init triggers
2022-02-02 17:33:32,894 INFO    MainThread:8677 [wandb_init.py:init():425] wandb.init called with sweep_config: {}
config: {}
2022-02-02 17:33:32,895 INFO    MainThread:8677 [wandb_init.py:init():471] starting backend
2022-02-02 17:33:32,895 INFO    MainThread:8677 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-02 17:33:32,916 INFO    MainThread:8677 [backend.py:ensure_launched():219] starting backend process...
2022-02-02 17:33:32,927 INFO    MainThread:8677 [backend.py:ensure_launched():224] started backend process with pid: 8720
2022-02-02 17:33:32,929 INFO    MainThread:8677 [wandb_init.py:init():480] backend started and connected
2022-02-02 17:33:32,934 INFO    MainThread:8677 [wandb_init.py:init():550] updated telemetry
2022-02-02 17:33:32,953 INFO    MainThread:8677 [wandb_init.py:init():581] communicating current version
2022-02-02 17:33:33,428 INFO    MainThread:8677 [wandb_init.py:init():586] got version response 
2022-02-02 17:33:33,428 INFO    MainThread:8677 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-02-02 17:34:03,429 INFO    MainThread:8677 [interface_shared.py:_communicate_run():313] couldn't get run from backend
2022-02-02 17:34:03,430 ERROR   MainThread:8677 [wandb_init.py:init():601] backend process timed out
2022-02-02 17:34:03,430 ERROR   MainThread:8677 [wandb_init.py:init():610] encountered error: Error communicating with wandb process
try: wandb.init(settings=wandb.Settings(start_method='fork'))
or:  wandb.init(settings=wandb.Settings(start_method='thread'))
For more info see: https://docs.wandb.ai/library/init#init-start-error
2022-02-02 17:34:03,453 ERROR   MainThread:8677 [wandb_init.py:init():978] error
Traceback (most recent call last):
  File "/home/yogender/miniconda3/envs/MLOps/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 951, in init
    run = wi.init()
  File "/home/yogender/miniconda3/envs/MLOps/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 614, in init
    backend.cleanup()
  File "/home/yogender/miniconda3/envs/MLOps/lib/python3.8/site-packages/wandb/sdk/backend/backend.py", line 248, in cleanup
    self.interface.join()
  File "/home/yogender/miniconda3/envs/MLOps/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 458, in join
    super().join()
  File "/home/yogender/miniconda3/envs/MLOps/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 599, in join
    _ = self._communicate_shutdown()
  File "/home/yogender/miniconda3/envs/MLOps/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 455, in _communicate_shutdown
    _ = self._communicate(record)
  File "/home/yogender/miniconda3/envs/MLOps/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 213, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/yogender/miniconda3/envs/MLOps/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 218, in _communicate_async
    raise Exception("The wandb backend process has shutdown")
Exception: The wandb backend process has shutdown
